lib/msal-node/src/client/ManagedIdentitySources/AzureArc.ts

```diff
@@ -168,10 +175,60 @@ export class AzureArc extends BaseManagedIdentitySource {
                 );
             }
 
-            const secretFile = wwwAuthHeader.split("Basic realm=")[1];
+            const secretFilePath = wwwAuthHeader.split("Basic realm=")[1];
+
+            // throw an error if the managed identity application is not being run on Windows or Linux
+            if (
```

Comment:
Strange spacing for paranthesis etc.

Reply:
That's done automatically by the "prettier" package we use to auto-formats code.